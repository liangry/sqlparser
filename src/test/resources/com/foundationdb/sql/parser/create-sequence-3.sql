CREATE SEQUENCE s START WITH 10 NO MINVALUE NO MAXVALUE NO CYCLE
