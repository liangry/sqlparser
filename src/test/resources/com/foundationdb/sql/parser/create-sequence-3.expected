com.foundationdb.sql.parser.CreateSequenceNode@2ddf191
name: s
statementType: CREATE SEQUENCE
sequenceName: s
dataType: null
startWith: 10
incrementBy: null
maxValue: null
minValue: null
isCycle: false
